service/mongodb/store.go

```diff
@@ -212,6 +218,43 @@ func (s MongoStore) GetAll(ctx context.Context, filter interface{}) ([]service.P
 	return permissions, nil
 }
 
+// GetUserPermissionsByPage returns a slice of the permissions requested by the filter,
```

Comment:
maybe add a documentation about the sortBy field with the MongoId

Reply:
done